diagnosis/README.md

```diff
@@ -19,6 +19,16 @@ The logs retrieved by `getkuberneteslogs.sh` are the following:
 - docker status and journal
 - kube-system snapshot
 
+Some additional logs are retrieved for windows nodes:
+
+ - Log file `c:\Azure\CustomDataSetupScript.log`
+ - kube-proxy status and journal
+ - containerd status and journal
+ - azure-vnet log and azure-vnet-telemetry log
+ - ETW events for docker
+ - ETW events for Hyper-V
+ - ETW events for some system events
```

Comment:
I would rather remove the this line if it does not include a specific detail

Reply:
Removed